app/utils/notification/index.ts

```diff
@@ -90,12 +90,27 @@ export const emitNotificationError = (type: 'Team' | 'Channel' | 'Post' | 'Conne
 
 export const scheduleExpiredNotification = (serverUrl: string, session: Session, serverName: string, locale = DEFAULT_LOCALE) => {
     const expiresAt = session?.expires_at || 0;
-    const expiresInDays = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asDays()));
+    const expiresInHours = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asHours()));
+    const expiresInDays = Math.floor(expiresInHours / 24); // Calculate expiresInDays
+    const remainingHours = expiresInHours % 24; // Calculate remaining hours
     const intl = createIntl({locale, messages: getTranslations(locale)});
-    const body = intl.formatMessage({
-        id: 'mobile.session_expired',
-        defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {daysCount, number} {daysCount, plural, one {day} other {days}}.',
-    }, {siteName: serverName, daysCount: expiresInDays});
+    let body = '';
+    if (expiresInDays === 0) {
+        body = intl.formatMessage({
+            id: 'mobile.session_expired_hrs',
+            defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {hoursCount, number} {hoursCount, plural, one {hour} other {hours}}.',
+        }, {siteName: serverName, hoursCount: remainingHours});
+    } else if (expiresInHours === 0) {
+        body = intl.formatMessage({
+            id: 'mobile.session_expired_days',
+            defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {daysCount, number} {daysCount, plural, one {day} other {days}}.',
+        }, {siteName: serverName, daysCount: expiresInDays, hoursCount: remainingHours});
```

Comment:
We no longer need the hours count, so we can remove it from here.

Reply:
Done